Automattic/newspack-plugin
# Remove all mentions of Muriel

### All Submissions:






### Changes proposed in this Pull Request:

This PR removes the remaining mentions of "Muriel" throughout the plugin:

* Update Package description and keywords
* Update Components readme
* Switch Wizard classes to `newspack-wizard` and `newspack-wizard__is-hidden`
* Remove unneeded divs and classNames
* Remove murielClassnames
* Remove _muriel-component stylesheet

### How to test the changes in this Pull Request:

1. Switch to this branch.
2. Check every pages. Do you notice anything broken?

### Other information:
